src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs

```diff
@@ -1196,5 +1196,44 @@ static void Goo()
 }
 ", "await", "class");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task AttributeNameAfterElementName()
+        {
+            var text = @"
+class C
+{
+    /// <exception $$
+    void Goo() { }
+}
+";
+            await VerifyItemExistsAsync(text, "cref");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task PartiallyTypedAttributeName()
```

Comment:
Technically, this test is redundant with the one above it because VerifyItemExistsAsync will test with the expected item partially typed (unless we override that behavior in xml doc tests?)